subprojects/test-runner/client/src/main/kotlin/com/avito/runner/scheduler/runner/TestRunnerImpl.kt

```diff
@@ -97,13 +106,13 @@ internal class TestRunnerImpl(
                     )
 
                     testSuiteListener.onTestSuiteFinished()
-                    logger.info("Test run end successfully")
+                    logger.info("Test run finished successfully")
                     Result.Success(result)
                 } catch (e: TimeoutCancellationException) {
-                    logger.critical("Test run end with timeout", e)
+                    logger.critical("Test run finished with timeout", e)
                     Result.Failure(e)
```

Comment:
Or we won't return this Failure when timeout occurs

Reply:
The current behavior isn't obvious. We return some Failure from the catch block but ignore it because of coroutine internals.
So I propose do not catch the TimeoutException and move logs to the creation of the default value

Reply:
Yes, that's true, but I don't know how to write it better

Reply:
> So I propose do not catch the TimeoutException and move logs to the creation of the default value

looks good to me

Reply:
☑